05-descriptive-analysis.Rmd

```diff
@@ -307,19 +309,19 @@ The difference between `survey_total()` and `survey_count()` is more evident whe
 recs_des %>%
   summarize(
     elec_bill = survey_total(DOLLAREL),
-    elec_unweight = unweighted(sum(!is.na(DOLLAREL))),
-    .groups = "drop"
+    elec_unweight = unweighted(sum(!is.na(DOLLAREL))),  
+    .groups = "drop"    
   )
 ```
 
-$162,495,237,023 was spent on electricity. The unweighted sum is $5686. <!--TODO: add calculation-->
+$162,495,237,023 was spent on electricity. The unweighted sum is $5686. <!--TODO: add calculation-->   
```

Comment:
This total is the national aggregate estimate. Fixed unweighted sum. Let's add note that unweighted sum is the total electricity expenditure among the *respondents*

Reply:
What's the timeframe for this?  Is it the amount spent in a year?  Would be helpful for some context.  Something to think about across all the sections as well.